### PR TITLE
ISPN-6868 Expose extra configuration for the SSLContext protocol in t…

### PR DESCRIPTION
**Pull request by gustavonalle**
…he Hot Rod client

https://issues.jboss.org/browse/ISPN-6868

### CODE REVIEW

File: commons/src/main/java/org/infinispan/commons/util/SslContextFactory.java [hunk 1]
```diff
@@ -24,12 +24,21 @@
  */
 public class SslContextFactory {
    private static final Log log = LogFactory.getLog(SslContextFactory.class);
+   public static final String DEFAULT_SSL_PROTOCOL = "TLS";
```
Comment (tristantarrant) [inline commons/src/main/java/org/infinispan/commons/util/SslContextFactory.java]: I think it would be best to increase this to TLSv1.2

Comment (gustavonalle) [inline commons/src/main/java/org/infinispan/commons/util/SslContextFactory.java]: Sure, I was with Java 1.6 in my mind when creating this PR  :smile:

**Comment by gustavonalle** [comment]
@tristantarrant fixed

**Comment by slaskawi** [comment]
The code looks good to me but could you please mention somewhere in the docs what it does and when to use it? Just a few words...

**Comment by tristantarrant** [comment]
I have pushed it already, so the docs will need to be in a separate PR

**Pull request closed without merging**